thorlcr/graph/thgraph.hpp

```diff
@@ -1045,12 +1051,12 @@ class graph_decl CActivityBase : public CInterface, implements IExceptionHandler
     virtual unsigned queryActivityId() { return (unsigned)queryId(); }
     virtual ICodeContext *queryCodeContext() { return container.queryCodeContext(); }
 
-    StringBuffer &getOpt(const char *prop, StringBuffer &out) const;
-    bool getOptBool(const char *prop, bool defVal=false) const;
-    int getOptInt(const char *prop, int defVal=0) const;
-    unsigned getOptUInt(const char *prop, unsigned defVal=0) const { return (unsigned)getOptInt(prop, defVal); }
-    __int64 getOptInt64(const char *prop, __int64 defVal=0) const;
-    unsigned __int64 getOptUInt64(const char *prop, unsigned __int64 defVal=0) const { return (unsigned __int64)getOptInt64(prop, defVal); }
+    StringBuffer &getOpt(const char *prop, StringBuffer &out) const { return container.getOpt(prop, out); }
+    bool getOptBool(const char *prop, bool defVal=false) const { return container.getOptBool(prop, defVal); }
+    int getOptInt(const char *prop, int defVal=0) const { return container.getOptInt(prop, defVal); }
+    unsigned getOptUInt(const char *prop, unsigned defVal=0) const { return (unsigned)container.getOptInt(prop, defVal); }
```

Comment:
minor: Should really call container.getOptUInt, and allow that to cast.  Same for getOptUInt64